frontend/crd/src/domain/Event.ts

```diff
@@ -19,14 +19,17 @@ export class Event {
     this.organizer = json.organizer;
     this.location = json.location;
     this.isRequired = json.isRequired;
```

Comment:
You might be able to replace this whole chunk with Object.assign from the JSON to the class, and then set the date afterwards.

Reply:
this only seems to somewhat work. It would require using a somewhat hacky workaround or else the compiler will complain about them being undefined